BDArmory/Ammo/PooledRocket.cs

```diff
@@ -910,8 +910,8 @@ public void BeehiveDetonation()
                     pBullet.bulletDmgMult = bulletDmgMult;
                     pBullet.ballisticCoefficient = sBullet.bulletMass / (((Mathf.PI * 0.25f * sBullet.caliber * sBullet.caliber) / 1000000f) * 0.295f);
                     pBullet.timeElapsedSinceCurrentSpeedWasAdjusted = 0;
-                    pBullet.timeToLiveUntil = 4000 / sBullet.bulletVelocity * 1.1f + Time.time;
-                    Vector3 firedVelocity = VectorUtils.GaussianDirectionDeviation(currentVelocity.normalized, (sBullet.subProjectileCount / BDAMath.Sqrt(currentVelocity.magnitude / 10))) * (sBullet.bulletVelocity / 10); //more subprojectiles = wider spread, higher base velocity = tighter spread
+                    pBullet.timeToLiveUntil = 2000 / sBullet.bulletVelocity * 1.1f + Time.time;
+                    Vector3 firedVelocity = VectorUtils.GaussianDirectionDeviation(currentVelocity.normalized, sBullet.subProjectileDispersion > 0 ? sBullet.subProjectileDispersion : (sBullet.subProjectileCount / BDAMath.Sqrt(currentVelocity.magnitude / 100))) * sBullet.bulletVelocity; //more subprojectiles = wider spread, higher base velocity = tighter spread
```

Comment:
Bringing the two `/ 10` together as `/ 100` isn't the same as the previous expression since one of them was inside the `Sqrt`, so it's being scaled by `1/10` now instead of `1/31.623`. Is this intentional?

Reply:
Yes. Increasing the divide by 10 to divide by 100 was to increase spread area slightly as compared to the earlier method, which was producing very tight groupings for low submunition counts, assuming a dispersion value isn't provided.
The removing the subMunition.bulletVelocity / 10 was to make that field more intuitive to configure

Reply:
OK, cool. I just wanted to check that it was intentional.